evalml/pipelines/components/estimators/classifiers/lightgbm_classifier.py

```diff
@@ -21,7 +21,7 @@ class LightGBMClassifier(Estimator):
         "boosting_type": ["gbdt", "dart", "goss", "rf"],
         "n_estimators": Integer(10, 100),
         "max_depth": Integer(0, 10),
-        "num_leaves": Integer(1, 100),
+        "num_leaves": Integer(2, 100),
```

Comment:
Ah, I see. So 1/100th of the time, we were setting `num_leaves` to 1 and the estimator was failing.

It would be great if we had coverage where we fit/transform/predict each component with the min and max allowed values from the defined hyperparameter ranges. Not required for this PR, but... let's keep it in mind :)

Reply:
Yeah, that makes sense! I'll keep that in mind